terraform/datadog/route-emitter.tf

```diff
@@ -90,3 +90,31 @@ resource "datadog_monitor" "route_emitter_lock_held_once" {
     "job"        = "route_emitter"
   }
 }
+
+resource "datadog_monitor" "route_emitter_fresh_registered_routes" {
+  name                = "${format("%s route-emitter has refreshed registered routes from BBS", var.env)}"
+  type                = "query alert"
+  message             = "route-emitter is not refreshing the routes with BBS"
+  escalation_message  = "route-emitter is still not refreshing the routes with BBS"
+  notify_no_data      = true
+  no_data_timeframe   = "7"
+  require_full_window = true
+
+  query = "${
```

Comment:
true, you are right. I played for a while and  Iwas not able to to implement anything. For instance, I tried a function that replaces nulls with 0, and then alert on count, but it does not work :(